jira.py

```diff
@@ -32,6 +38,82 @@ def __init__(self):
     # API: Filters
     def filters(self):
         query = JQL_QUERY + FILTER_ID_ALL_REQUESTS_2022 + '&fields=' \
-                + DEFAULT_COLUMNS + STORY_POINTS + FIREFOX_RELEASE_TRAIN \
```

Comment:
same as above.  We should consider if there will be a number of JQL queries then perhaps we should move them either to constants or to a file called queries_jql.py.  Each query could have a constant name.